prerequisites/check-version.js

```diff
@@ -0,0 +1,28 @@
+var semver = require("semver");
+var requiredVersion = require("../package.json").engines.node;
+
+function requiredToList(requiredVersion) {
+	return requiredVersion.split("||").filter(Boolean).map(function(version) {
```

Comment:
The range object provides this splitting functionality for us, so we should be able to get away with:

```js
new semver.Range(requiredVersion).set.map(function(set) {
  // ...
});
```